src/ontology/edits/oeo-physical.omn

```diff
@@ -1607,6 +1676,16 @@ Class: NaturalGasGenerator
         FossilGenerator
     
     
+Class: NegativeEmission
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "NegativeEmission is the process of absorbing a substance (that was emitted before, typically a pollution).",
```

Comment:
Having a part of the definition text enclosed in brackets makes it seem a bit uncommitted. Also I don't think it is possible to say "a pollution". How about: 
"NegativeEmission is the process of absorbing a substance, usually a pollutant that was emitted before."